src/languages/es.ts

```diff
@@ -2258,6 +2259,21 @@ export default {
             existingConnections: 'Conexiones existentes',
             lastSyncDate: (connectionName: string, formattedDate: string) => `${connectionName} - Última sincronización ${formattedDate}`,
             topLevel: 'Nivel superior',
+            memberAlternateText: 'Los miembros pueden presentar y aprobar informes.',
```

Comment:
Did we confirm these translations? I feel like I saw it in channel but I couldn't find it when I just searched for it...

Reply:
Sorry I forgot to request translation.
Requested on slack here: https://expensify.slack.com/archives/C01GTK53T8Q/p1725938247021149